README.md

```diff
@@ -66,13 +66,13 @@ the platform.
 A Dockerfile is provided in `./cross_build` that demonstrates how to cross compile the example application and all dependencies. The example build is for `aarch64`. To adopt to a custom toolchain, adjust the Dockerfile to include and use the custom toolchain. Assuming the custom toolchain is available in "camera-toolchain.tar.gz", it can be installed it into the image by modifying the Dockerfile as follows:
 
 ```
-RUN apt-get update && apt-get install git build-essential cmake gcc-aarch64-linux-gnu g++-aarch64-linux-gnu curl file tar -y
```

Comment:
With the aarch64 example, this demonstrates that you have to install the appropriate compilers for your architecture. To me it seems like a better example for the docs than armv4 which apparently does not have this requirement

Reply:
We have yet to see a single customer ever being to install a relevant cross compiler through a package manager, hence the example with the toolchain in a tar file.